brave/brave-core
# Adds balance info to internals page

Resolves https://github.com/brave/brave-browser/issues/9512

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim promotions
- connect KYC uphold account
- go to internals page
- make sure that you see correct balance there

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
